shared/bash_remediation_functions/fix_audit_syscall_rule.sh

```diff
@@ -89,16 +89,17 @@ then
 	fi
 	for match in "${matches[@]}"
 	do
-		files_to_inspect=("${files_to_inspect[@]}" "${match}")
+		files_to_inspect+=("${match}")
 	done
 	# Case when particular rule isn't defined in /etc/audit/rules.d/*.rules yet
 	if [ ${#files_to_inspect[@]} -eq "0" ]
 	then
-		files_to_inspect="/etc/audit/rules.d/$key.rules"
```

Comment:
`files_to_inspect` is used below as an array, so let's make it an array that has length of 1. Before, it was redefined as a variable.